README.md

```diff
@@ -21,8 +45,8 @@ Run the following command to install OpenEBS velero-plugin
 
 This command will add an init container to Velero deployment to install the OpenEBS velero-plugin.
```

Comment:
In this section, first sentence is providing the step to install velero-plugin with ci image. So you can mention like this was "install development image of velero-pugin".